packages/integrations/cloudflare/test/cf.test.js

```diff
@@ -9,16 +9,20 @@ describe('Cf metadata and caches', () => {
 	/** @type {import('./test-utils').WranglerCLI} */
 	let cli;
 
-	before(async () => {
+	before(async function () {
 		fixture = await loadFixture({
 			root: './fixtures/cf/',
 			output: 'server',
 			adapter: cloudflare(),
 		});
 		await fixture.build();
 
-		cli = await runCLI('./fixtures/cf/', { silent: false, port: 8788 });
-		await cli.ready;
+		cli = await runCLI('./fixtures/cf/', { silent: true, port: 8786 });
```

Comment:
Changed the port number as it's close to the other test's `8789` port, which maybe was causing issues somehow.